src/views/CampusSafety/views/LostAndFound/views/ReportFound/index.tsx

```diff
@@ -0,0 +1,55 @@
+import { Button, Card, CardContent, CardHeader, Grid, Typography } from '@mui/material';
+import { LocalLibrary, LocalPolice } from '@mui/icons-material';
+import { useNavigate } from 'react-router';
+import Header from 'views/CampusSafety/components/Header';
+
+const ReportFound = () => {
+  const navigate = useNavigate();
+
+  return (
+    <>
+      <Header />
+      <Card>
```

Comment:
This card should probably be centered and sized relative to the content, but I assume you're working on that.

Reply:
Fixed, good point!